src/checkfiles/checkfiles.py

```diff
@@ -394,6 +399,23 @@ def vcf_sequence_check(file_path, assembly):
     return error
 
 
+def seqspec_file_check(file_path):
+    error = {}
+    # check if IGVF_API_KEY and IGVF_SECRET_KEY are set
+    if 'IGVF_API_KEY' not in os.environ or 'IGVF_SECRET_KEY' not in os.environ:
+        logger.warning(
+            f'IGVF_API_KEY and IGVF_SECRET_KEY are not set. seqspec check will not be able to access files that are not released.')
+    try:
+        # validate seqspec yaml file without upgrading
+        spec = seqspec_load_spec(file_path)
+        errors = seqspec_check(spec, file_path, for_igvf=True)
+        if errors:
+            error['seqspec_error'] = errors
+    except Exception as e:
```

Comment:
Can we be more specific about the `Execption`s we expect? Here we might end up catching something that is not really relevant to the actual check.

Reply:
If we don't know, just add logging so that if we are ever confused about it, at least it will be easier to find.